rfcs/2020-09-21-0000-kubernetes-metrics-overview.md

```diff
@@ -0,0 +1,283 @@
+# RFC 0000 - 2020-09-21 - Overview of metrics in Kubernetes
+
+This RFC's goal is to create a systematic overview of metrics in the
+Kubernetes ecosystem.
+
+## Table of contents
+
+- [RFC 3684 - 2020-09-21 - Overview of metrics in Kubernetes](#rfc-3684---2020-09-21---overview-of-metrics-in-kubernetes)
+  - [Table of contents](#table-of-contents)
+  - [Motivation](#motivation)
+  - [Overview](#overview)
+  - [Metrics exposed by the Kubernetes system components](#metrics-exposed-by-the-kubernetes-system-components)
+    - [Properties](#properties)
+    - [Per-component insights](#per-component-insights)
+  - [TODO](#todo)
+  - [Prior art](#prior-art)
+
+## Motivation
+
+Metrics is Kubernetes are a large sector, with many concerns. We need a way to
+better understand it, to, sort of, have a list of entry points to explore in
+more detail.
+
+This overview will help us reason about the metrics  in Kubernetes and make
+educated design and planning decisions.
+
+## Overview
+
+- Application metrics exposed via [OpenMetrics](https://openmetrics.io/) or
```

Comment:
This is Vector's job, and operators will want to configure the appropriate sources to pull those metrics. Your StatsD examples is exactly where Vector shines. Vector can ingest those metrics without application-level changes.